srv/modules/runners/validate.py

```diff
@@ -593,21 +616,25 @@ def pillars(**kwargs):
     #options = SaltOptions()
     local = salt.client.LocalClient()
     cluster = ClusterAssignment(local)
+    
+    printer = printer = get_printer(**kwargs)
+
 
     for name in cluster.names:
-        pillar(name, **kwargs)
+        pillar(name, printer=printer, **kwargs)
+    
+    printer.print_result()
 
 
-def pillar(name = None, **kwargs):
+def pillar(name = None, printer=None, **kwargs):
     """
     Check that the pillar for each cluster meets the requirements to install
     a Ceph cluster.
     """
-
-    if name == None:
-        if 'name' in kwargs:
-            name = kwargs['name']
-
```

Comment:
Are you worried that somebody will call validate with

salt-run validate.pillar ceph name=ceph

I get what you expect

TypeError: pillar() got multiple values for keyword argument 'name'

which is what I would expect.  I'm not sure what expectations of running the command I'm hoping to avoid.